gtsam/hybrid/tests/testHybridGaussianISAM.cpp

```diff
@@ -177,10 +178,10 @@ TEST(HybridGaussianElimination, IncrementalInference) {
 
   // Test if the probability values are as expected with regression tests.
```

Comment:
"as expected" and "regression test" are mutually incompatible. If this is a unit test, say how you computed 0.1666667. If it is a regression tests, we'll assume you copied from output to safeguard against future regressions.